### PR TITLE
Fix primary particle definition in CheckTracks.C

### PR DESCRIPTION
**Pull request by mpuccio**
In PYTHIA events, primary particles come from quarks not from a mother with id -1

### CODE REVIEW

**Comment by mpuccio** [comment]
@iouribelikov @shahor02 have a look when you have time, I found this small issue after the recent changes in the stack

File: Detectors/ITSMFT/ITS/macros/test/CheckTracks.C [hunk 1]
```diff
@@ -305,8 +305,12 @@ void CheckTracks(std::string tracfile = "o2trac_its.root", std::string clusfile
...
-      if (mID >= 0)
-        continue; // Select primary particles
+      if (mID >= 0) {
+        const auto& mom = (*mcArr)[mID];
+        int pdg = std::abs(mom.GetPdgCode());
+        if (pdg > 100 || (pdg < 20 && pdg > 10))
+          continue; // Select primary particles
+      }
       Int_t pdg = mcTrack.GetPdgCode();
       if (TMath::Abs(pdg) != 211)
```
Comment (shahor02) [inline Detectors/ITSMFT/ITS/macros/test/CheckTracks.C:315]: in this case, should not this be eliminated

File: Detectors/ITSMFT/ITS/macros/test/CheckTracks.C [hunk 2]
```diff
@@ -305,8 +305,12 @@ void CheckTracks(std::string tracfile = "o2trac_its.root", std::string clusfile
       }
 
       Int_t mID = mcTrack.getMotherTrackId();
-      if (mID >= 0)
-        continue; // Select primary particles
+      if (mID >= 0) {
+        const auto& mom = (*mcArr)[mID];
+        int pdg = std::abs(mom.GetPdgCode());
+        if (pdg > 100 || (pdg < 20 && pdg > 10))
```
Comment (shahor02) [inline Detectors/ITSMFT/ITS/macros/test/CheckTracks.C:311]: but what about the primary electrons and muons? 
Right now we don't really select the primaries.

File: Detectors/ITSMFT/ITS/macros/test/CheckTracks.C (hunk 1 above)
Comment (mpuccio) [reply Detectors/ITSMFT/ITS/macros/test/CheckTracks.C:315]: The check on pions? We can remove it

File: Detectors/ITSMFT/ITS/macros/test/CheckTracks.C (hunk 2 above)
Comment (mpuccio) [reply Detectors/ITSMFT/ITS/macros/test/CheckTracks.C:311]: Here just the daughters of electrons and muons are rejected, electrons and muons themself are not touched by this selection. Am I missing something?

**Pull request merged**